strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/ProxyRepositoryProviderTestIT.java

```diff
@@ -49,19 +53,33 @@
     @Inject
     private ArtifactResolutionService artifactResolutionService;
 
+    @Inject
+    private ArtifactEntryService artifactEntryService;
+
     @Inject
     private MavenMetadataManager mavenMetadataManager;
 
     @Before
-    public void setUp()
+    @After
+    public void cleanup()
             throws Exception
     {
-        File derbyPluginBaseDir = new File(ConfigurationResourceResolver.getVaultDirectory() +
-                                           "/storages/storage-common-proxies/maven-central/" +
-                                           "org/carlspring/maven/derby-maven-plugin");
-        if (derbyPluginBaseDir.exists())
+
```

Comment:
As a rule of thumb, it's always safer to use paths that don't start with `/`, especially under Linux/Unix, if you know that there will be a base path prepended. The reason is that -- in case of a minor bug in your code (and we are all human), if this path for some reason matches an existing directory under the root of the file system, you can wipe out stuff that you really didn't mean to and cause a massive mess... (I'm not saying that there is an issue with your code, I'm just trying to point out why it's not good practice).

Would you mind changing this? :)

Reply:
@carlspring , please notice that this is just an argument to the newly created method `ProxyRepositoryProviderTestIT#deleteDirectory` which concatenates `ConfigurationResourceResolver.getVaultDirectory()` with the provided argument. So the full path is for example `ConfigurationResourceResolver.getVaultDirectory() + "/storages/storage-common-proxies/maven-central/org/carlspring/maven/derby-maven-plugin"`. However, thanks I'll add one additional check for that :)